### PR TITLE
add contract test for parking status check

### PR DESCRIPTION
**Pull request by diegolopezgrisanti**
## Trello ticket
[Ticket](https://trello.com/c/W1wrrAe8/41-add-contract-tests)

## Description of changes

- Add contract test for checkParking

### CODE REVIEW

File: src/test/java/com/parkingapp/parkingservice/infrastructure/entrypoint/rest/ParkingControllerContractTest.java [hunk 1]
```diff
@@ -262,6 +262,31 @@ private MockMvcResponse whenARequestToGetParkingByIdIsReceived() {
     @Nested
     class CheckParking {
 
+        @Test
+        void shouldReturnParkingAndStatus() {
```
Comment (solblack) [inline src/test/java/com/parkingapp/parkingservice/infrastructure/entrypoint/rest/ParkingControllerContractTest.java:266]: 2 tests that include body asserts:

void shouldReturnParkingAndStatusWithParkingDetail() {

void shouldReturnParkingAndStatusWithNoParkingDetail() {

File: src/test/java/com/parkingapp/parkingservice/infrastructure/entrypoint/rest/ParkingControllerContractTest.java [hunk 2]
```diff
@@ -262,6 +262,31 @@ private MockMvcResponse whenARequestToGetParkingByIdIsReceived() {
...
+            // GIVEN
+            when(checkParkingStatusUseCase.execute(plate, parkingZoneId)).thenReturn(parkingStatusCheck);
+
+            // WHEN
+            MockMvcResponse response = whenARequestToGetParkingStatusIsReceived();
+
+            // THEN
+            response.then()
+                    .statusCode(HttpStatus.OK.value());
```
Comment (solblack) [inline src/test/java/com/parkingapp/parkingservice/infrastructure/entrypoint/rest/ParkingControllerContractTest.java:275]: missing body assertion

File: src/test/java/com/parkingapp/parkingservice/infrastructure/entrypoint/rest/ParkingControllerContractTest.java (hunk 1 above)
Comment (fabriciodigennaro) [reply src/test/java/com/parkingapp/parkingservice/infrastructure/entrypoint/rest/ParkingControllerContractTest.java]: done! 👍🏻

File: src/test/java/com/parkingapp/parkingservice/infrastructure/entrypoint/rest/ParkingControllerContractTest.java (hunk 2 above)
Comment (fabriciodigennaro) [reply src/test/java/com/parkingapp/parkingservice/infrastructure/entrypoint/rest/ParkingControllerContractTest.java]: done 👍🏻 !

**Review by solblack — APPROVED** [review]

**Review by solblack — APPROVED** [review]

**Pull request merged**